DOC-756: Document new table row numbering (series column) advtable feature (#2086)
tinymce/tinymce-docs: _includes/commands/advtable-cmds.md

| Command                  | Description                                                                                     |
| ------------------------ | ----------------------------------------------------------------------------------------------- |
| mceAdvancedTableSort     | Opens the Advanced Table Sort Dialog for the current selection or cursor location.              |

> | mceSortTableAdvanced     | Performs an Advanced Table Sort. For details, see [mceSortTableAdvanced](#mcesorttableadvanced).                                         |

Reviewer: ```suggestion
| mceSortTableAdvanced     | Performs an Advanced Table Sort. For details, see [`mceSortTableAdvanced`](#mcesorttableadvanced).                                         |
```

> | mceSortTableAdvanced     | Performs an Advanced Table Sort. For details, see [`mceSortTableAdvanced`](#mcesorttableadvanced).                                         |

Reviewer 2: ```suggestion
| mceSortTableAdvanced     | Performs an Advanced Table Sort. For details, see [Using `mceSortTableAdvanced`](#usingmcesorttableadvanced).                                         |
```

Reviewer 2: changed these because they looked a bit like a circular reference.

| mceSortTableByColumnAsc  | Sorts the current table ascending by column based on the current cursor position or selection.  |
| mceSortTableByColumnDesc | Sorts the current table descending by column based on the current cursor position or selection. |
| mceSortTableByRowAsc     | Sorts the current table ascending by row based on the current cursor position or selection.     |
| mceSortTableByRowDesc    | Sorts the current table descending by row based on the current cursor position or selection.    |

> | mceTableToggleSeries     | Toggles a series column on the selected table. For details, see [`mceTableToggleSeries`](#mcetabletoggleseries). {{ site.requires_5_9v }} |

Reviewer 2: ```suggestion
| mceTableToggleSeries     | Toggles a series column on the selected table. For details, see [Using `mceTableToggleSeries`](#usingmcetabletoggleseries). {{ site.requires_5_9v }} |
```

tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numeric', location: 'left' })
```

### `mceSortTableAdvanced`

`mceSortTableAdvanced` accepts an object with the following key-value pairs:

> - `sortby` - `'row'` or `'column'`

Author: Happy to do that if Tyler thinks it is suitable.

> - `sortby` - `'row'` or `'column'`

Author: I have made it a table to be more consistent

- `roworcol` - a zero-indexed integer in a string representing the row from the top of the table or column from the left of the table.
- `sort` - `'row'`, `'column'`, `'selection'`, or `'table'`
- `order` - `'ascending'` or `'descending'`

### `mceTableToggleSeries`

`mceTableToggleSeries` accepts an object with the following key-value pairs:

| Name | Value | Requirement | Description |
| ---- | ----- | ----------- | ----------- |

> | name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). If the series name provided is not present on the table, a series column will be inserted or an existing series column will be replaced by the new series. |

Reviewer 3: I think this is right?

```suggestion
| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). If a series column with the series name provided is not present on the table, a series column will be inserted or an existing series column will be replaced by the new series. |
```

Should we explain what happens if there is already a series column with the same name?

Author: I am wondering if doing something like this would be too verbose?

```
| Name | Value | Requirement | Description |
| ---- | ----- | ----------- | ----------- |
| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [`advtable_value_series` option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). |

If a series column is not present on a table:
- a series column using the specified series will be inserted at the start of the table.

If a series column is present on a table:
- if the specified series matches, the series column is removed from the table.
- if the specified series does not match, the series column is updated to use the new series.
```

Reviewer 4: I like taking the toggling information out of the table, but maybe structure it more like this?

```suggestion
| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). |

If the table already has a series column that uses the series specified in `name`, the series column will be removed from the table. Otherwise, a new series column will be created, replacing any other series column that may already be in the table.
```

Author: Yep, I like that suggestion so will use that. Thanks.

> | location | `'left'` | Required | Specifies the location of where the series is toggled in the table. For example, a location of `left` will toggle a series column on the left side of the table.  |

Reviewer 3: Should this be "`'left'` or `'right'`"

Reviewer: Actually should just even be included at all? This is just something we put as a placeholder and definitely isn't required. As such, I believe this should be removed from the docs for now.

Author: That's a good point. I will remove `location` which should also make it less confusing.

tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numeric' })
```

> ### `mceSortTableAdvanced`

Reviewer 2: ```suggestion
### Using `mceSortTableAdvanced`
```

`mceSortTableAdvanced` accepts an object with the following key-value pairs:

| Name | Value | Requirement | Description |
| ---- | ----- | ----------- | ----------- |
| sortby | `'row'` or `'column'` | Required | |
| roworcol | `number` | Required | A zero-indexed integer in a string representing the row from the top of the table or column from the left of the table. |
| sort | `'row'`, `'column'`, `'selection'`, or `'table'` | Required | |
| order | `'ascending'` or `'descending'` | Required | |

> ### `mceTableToggleSeries`

Reviewer 2: ```suggestion
### Using `mceTableToggleSeries`
```
